releases/release-5.2.0.md

```diff
@@ -0,0 +1,339 @@
+|`tidb_enable_streaming`|废弃|废弃 TiDB 配置文件中`enable-streaming`配置项|
+
+### 配置文件参数
+
+| 配置文件   | 配置项   | 修改类型   | 描述   |
+|:----------|:-----------|:-----------|:-----------|
+|TiDB 配置文件|[`pessimistic-txn.deadlock-history-collect-retryable`](/tidb-configuration-file.md#deadlock-history-collect-retryable)|新增|控制 [`INFORMATION\_SCHEMA.DEADLOCKS`](/information-schema/information-schema-deadlocks.md) 表中是否收集可重试的死锁错误信息。|
+|TiDB 配置文件|[`security.auto-tls`](/tidb-configuration-file.md#auto-tls)|新增|控制 TiDB 启动时是否自动生成 TLS 证书，默认值为 `true`。|
+|TiDB 配置文件|[`stmt-summary.max-stmt-count`](/tidb-configuration-file.md#max-stmt-count)|修改|表示系统表 `statements_summary` 保存的 SQL 种类的最大数量。默认值从 200 修改为 3000。|
+|TiDB 配置文件|`experimental.allow-expression-index` |废弃|废弃 TiDB 配置文件中`allow-expression-index` 配置项|
+|TiKV 配置文件|[`inspect-interval`](/tikv-configuration-file.md#inspect-interval) |新增|TiKV 每隔一段时间会检测 Raftstore 线程的延迟情况，该配置项设置检测的时间间隔。默认值为 `500ms`。|
+|TiKV 配置文件|[`storage.flow-control.enable`](/tikv-configuration-file.md#enable) |新增|表示是否开启 TiKV 流量控制机制。默认值为 `true`，代表开启。|
+|TiKV 配置文件|[`storage.flow-control.memtables-threshold`](/tikv-configuration-file.md#memtables-threshold) |新增|当 KvDB 的 memtable 的个数达到该阈值时，流控机制开始工作。默认值为 5。|
+|TiKV 配置文件|[`storage.flow-control.l0-files-threshold`](/tikv-configuration-file.md#l0-files-threshold) |新增|当 KvDB 的 L0 文件个数达到该阈值时，流控机制开始工作。默认值为 9。|
+|TiKV 配置文件|[`storage.flow-control.soft-pending-compaction-bytes-limit`](/tikv-configuration-file.md#soft-pending-compaction-bytes-limit) |新增|当 KvDB 的 pending compaction bytes 达到该阈值时，流控机制开始拒绝部分写入请求并报错。默认值为 "192GB"。|
+|TiKV 配置文件|[`storage.flow-control.hard-pending-compaction-bytes-limit`](/tikv-configuration-file.md#hard-pending-compaction-bytes-limit) |新增|当 KvDB 的 pending compaction bytes 达到该阈值时，流控机制开始拒绝所有写入请求并报错。默认值为 "1024GB"。|
+
+### 其他
+
+- 升级前，请检查系统变量 `tidb_evolve_plan_baselines` 是否是 ON。如果是 ON，需要
+要改成 `OFF`，否则会导致升级失败。
+- v4.0 集群升级到 v5.0 或更高版本（dev 和 v5.1）的集群后，`tidb_multi_statement_mode` 变量的默认值由 `WARN` 变为 `OFF`。
+- 兼容 MySQL 5.7 的 noop 变量 `innodb_default_row_format`，配置此变量无实际效果 [#23541](https://github.com/pingcap/tidb/issues/23541)。
+- 升级至 5.2 版本时，TiKV 会通过`block-cache.capacity`计算`memory-usage-limit`，此时 `memory-usage-limit` 的计算结果默认为 “`block-cache.capacity` / 0.45 * 0.75”的值。
+
+    例如，当[storage.block-cache] capacity = “24GB”时，`memory-usage-limit`为 40GB (24GB/0.45*0.75 = 40GB)。
+
+- 表达式索引兼容性说明
+    - 升级过程中不允许执行表达式索引的 DDL 操作
+    - 通过视图进行查询时，无法使用表达式索引。
+    - 表达式索引与 Binding 存在兼容性问题。当表达式索引中的表达式存在常量时，对应的查询所建的 binding 会扩大范围。假设表达式索引的表达式为 `a+1`，对应的查询为 `a+1 > 2`。则建立的 binding 为 `a+? > ?`，这会导致像 `a+2 > 2` 这样的查询也会强制走表达式索引，得到一个较差的执行计划。这同样影响 SPM 中的捕获和演进功能。
+
+## 新功能
+
+### SQL
+
+- **新增表达式索引 (Expression index)**
+
+    表达式索引是一种特殊的索引，能将索引建立于表达式上。创建了表达式索引后，TiDB 支持基于表达式的查询，极大提升查询的性能。
+
+    [用户文档](/sql-statements/sql-statement-create-index.md)，[#25150](https://github.com/pingcap/tidb/issues/)
```

Comment:
```suggestion
    [用户文档](/sql-statements/sql-statement-create-index.md)，[#25150](https://github.com/pingcap/tidb/issues/25150)
```

Reply:
This issue doesn't provide any useful information to users, so I suggest removing it.